testgrid/config/config.yaml

```diff
@@ -1677,6 +1677,13 @@ test_groups:
   num_columns_recent: 30
 - name: tf-k8s-postsubmit
   gcs_prefix: kubernetes-jenkins/logs/tensorflow_k8s/tf-k8s-postsubmit
+- name: tf-minigo-periodic
+  gcs_prefix: kubernetes-jenkins/logs/tf-minigo-periodic
```

Comment:
and you didn't add a periodic job in prow

Reply:
Done